stackit/internal/services/argus/argus_acc_test.go

```diff
@@ -39,8 +42,9 @@ var credentialResource = map[string]string{
 	"project_id": testutil.ProjectId,
 }
 
-func resourceConfig(instanceName, planName, target, saml2EnableUrlParameters string) string {
-	return fmt.Sprintf(`
+func resourceConfig(acl *string, instanceName, planName, target, saml2EnableUrlParameters string) string {
```

Comment:
you set `acl` as a pointer but there is no test case where it is passed as `nil`, so it seems unnecessary complexity

Reply:
This a good point, but there should probably be an acceptance test to make sure it still works if no ACL is provided. My commit below adds this